native/packages/react-native-bpk-component-star-rating/readme.md

```diff
@@ -0,0 +1,45 @@
+# react-native-bpk-component-star-rating
+
+> Backpack React Native star rating component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-star-rating --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import StarRating from 'react-native-bpk-component-star-rating';
+import { spacingBase } from 'bpk-tokens/tokens/base.react.native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: spacingBase,
+  }
+});
+
+export default class App extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <StarRating rating={0} />
+        <StarRating rating={5} />
```

Comment:
We should have examples of `maxRating` and `ratingLabel` here too, I think.

Reply:
Yeah you're right!